tests/test_dates.py

```diff
@@ -250,6 +250,129 @@ def test_with_float(self):
         formatted_string = dates.format_datetime(epoch, format='long', locale='en_US')
         self.assertEqual(u'April 1, 2012 at 3:30:29 PM +0000', formatted_string)
 
+    def test_with_pattern(self):
```

Comment:
This seems to only be testing time zone formats, so rename to, say, `test_timezone_formats`? :)